stable/drupal/templates/ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+ name: {{ template "fullname" . }}
+ labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+ annotations:
+   {{- range $key, $value := .Values.ingress.annotations }}
+     {{ $key }}: {{ $value | quote }}
+   {{- end }}
+spec:
+  rules:
+    - host: {{ .Values.ingress.hostname }}
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: {{ template "fullname" . }}
+              servicePort: 80
```

Comment:
Are rules needed here? If not would spare users from setting `ingress.hostname` e.g. I've used the following ingress definition before:

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: {{ .Values.ingress.addressName }}
  annotations:
    kubernetes.io/ingress.global-static-ip-name: {{ .Values.ingress.addressName | quote }}
spec:
  backend:
    serviceName: {{ .Values.ingress.serviceName }}
    servicePort: {{ .Values.ingress.servicePort }}
  tls:
    - secretName: {{ .Values.ingress.tls.secretName }}
```